web-components/src/components/tabs/Tabs.ts

```diff
@@ -875,6 +873,17 @@ export namespace Tabs {
 
       this.compUniqueId && (this.tabsOrderPrefsArray = localStorage.getItem(this.compUniqueId)?.split(",") || []);
     }
+
+    private allElements: Array<String> = [];
+
+    private updateSelectedTabOnVoiceOver(e: any) {
```

Comment:
function name should not have tool name like voice over or JAWS, name should convey the purpose of the function

e.g. updateSelectedTabIndexOnClick